AddressBook_/AddressBook_/Controller/ViewController.swift

```diff
@@ -0,0 +1,247 @@
+            $0.bottom.equalTo(mainTableView.snp.top).offset(-15)
+        }
+        
+        addMemberButton.snp.makeConstraints {
+            $0.width.equalTo(60)
+            $0.top.equalTo(textFieldStackView.snp.bottom).offset(20)
+            $0.leading.equalTo(buttonStackView.snp.trailing).offset(45)
+//            $0.trailing.equalTo(testButton.snp.leading).offset(-10)
+            $0.bottom.equalTo(mainTableView.snp.top).offset(-15)
+        }
+        
+        testButton.snp.makeConstraints {
+            $0.width.equalTo(60)
+            $0.top.equalTo(textFieldStackView.snp.bottom).offset(20)
+            $0.leading.equalTo(addMemberButton.snp.trailing).offset(10)
+            $0.trailing.equalTo(view.snp.trailing).offset(-20)
+            $0.bottom.equalTo(mainTableView.snp.top).offset(-15)
+        }
+        
+        // 테이블 뷰
+        mainTableView.snp.makeConstraints {
+            $0.top.equalTo(buttonStackView.snp.bottom).offset(15)
+            $0.leading.equalTo(view.snp.leading).offset(20)
+            $0.trailing.equalTo(view.snp.trailing).offset(-20)
+            $0.bottom.equalTo(view.safeAreaLayoutGuide.snp.bottom).offset(-20)
+        }
+    }
+    
+    func addAction() {
+        allCategoryButton.addTarget(self, action: #selector(allCategoryButtonClicked), for: .touchUpInside)
+        favoriteCategoryButton.addTarget(self, action: #selector(favoriteCategoryButtonClicked), for: .touchUpInside)
+        addMemberButton.addTarget(self, action: #selector(addMemberButtonClicked), for: .touchUpInside)
+        testButton.addTarget(self, action: #selector(testButtonClicked), for: .touchUpInside)
+    }
+    
+    @objc func allCategoryButtonClicked() {
+        allCategoryButton.isSelected.toggle()
+        favoriteCategoryButton.isSelected = !allCategoryButton.isSelected
+        if allCategoryButton.isSelected {
+            allCategoryButton.backgroundColor = .systemBlue
+            favoriteCategoryButton.backgroundColor = .white
+        } else {
+            favoriteCategoryButton.backgroundColor = .systemBlue
+            allCategoryButton.backgroundColor = .white
+        }
+    }
+    
+    @objc func favoriteCategoryButtonClicked() {
+        favoriteCategoryButton.isSelected.toggle()
+        allCategoryButton.isSelected = !favoriteCategoryButton.isSelected
+        if allCategoryButton.isSelected {
+            allCategoryButton.backgroundColor = .systemBlue
+            favoriteCategoryButton.backgroundColor = .white
+        } else {
+            favoriteCategoryButton.backgroundColor = .systemBlue
+            allCategoryButton.backgroundColor = .white
+        }
+    }
+    
+    @objc func addMemberButtonClicked() {
+        let detailVC = DetailViewController()
+        detailVC.delegate = self
+        detailVC.modalPresentationStyle = .fullScreen
+        present(detailVC, animated: true)
+    }
+    
+    @objc func testButtonClicked() {
```

Comment:
내부 코드가 비어있는 함수의 경우 아래와 같이 작성해주세요
```suggestion
    @objc func testButtonClicked() { }
```